apple/Elements/RNSVGImage.mm

```diff
@@ -25,13 +24,107 @@
 
 #import <React/RCTLog.h>
 #import "RNSVGViewBox.h"
+#import "RCTBridge.h"
+
+#ifdef RN_FABRIC_ENABLED
+#import <react/renderer/components/rnsvg/ComponentDescriptors.h>
+#import "RCTFabricComponentsPlugins.h"
+#import "RCTConversions.h"
+#import <react/renderer/components/view/conversions.h>
+#import "RNSVGFabricConversions.h"
+#import "RCTImagePrimitivesConversions.h"
+#import "RCTImageSource.h"
+
+// Some RN private method hacking below similar to how it is done in RNScreens:
+// https://github.com/software-mansion/react-native-screens/blob/90e548739f35b5ded2524a9d6410033fc233f586/ios/RNSScreenStackHeaderConfig.mm#L30
+@interface RCTBridge (Private)
++ (RCTBridge *)currentBridge;
+@end
+
+#endif
```

Comment:
Just a style thing. Up to you. 
Personally I like when `#endif` is annotated with with information "what if statement it ends". It improves readability in long run & helps to navigate the code especially when there are some nested ifs. 

```
#ifdef RN_FABRIC_ENABLED
...
#endif // RN_FABRIC_ENABLED
```